src/converter.js

```diff
@@ -148,19 +149,16 @@ var getQueryStrings = function(swagger, path, method, values) {
 
   if (typeof swagger.paths[path][method].parameters !== 'undefined') {
     for (var i in swagger.paths[path][method].parameters) {
-      var param = swagger.paths[path][method].parameters[i]
+      var param = swagger.paths[path][method].parameters[i];
       if (typeof param['$ref'] === 'string' &&
         !/^http/.test(param['$ref'])) {
         param = resolveRef(swagger, param['$ref'])
       }
       if (typeof param.in !== 'undefined' && param.in.toLowerCase() === 'query') {
+        const sample = OpenAPISampler.sample(param.schema || param, {});
```

Comment:
You should pass `definitions` as third argument into `.sample()`. Otherwise, sample can't find schemes by json pointer.